tests/integration/models/api-key.spec.ts

```diff
@@ -124,6 +124,33 @@ describe('API Key model', function () {
 		});
 	});
 
+	describe('balena.models.apiKey.getProvisioningApiKeysByApplication', function () {
+		givenLoggedInUser(before);
+
+		before(function () {
+			this.appName = 'SampleOrg/sampleApp';
+			return balena.models.application
+				.generateApiKey('SampleOrg/sampleApp')
+				.then(function (apiKey) {
+					expect(_.isString(apiKey)).to.be.true;
+					expect(apiKey).to.have.length(32);
+
+					this.apiKey = apiKey;
+				});
+		});
+
+		describe('balena.models.apiKey.getProvisioningApiKeysByApplication()', function () {
+			return balena.models.apiKey
+				.getProvisioningApiKeysByApplication(this.appName)
+				.then(function (apiKeys) {
+					expect(apiKeys).to.be.an('array');
+					expect(apiKeys).to.have.lengthOf(1);
+					expect(apiKeys[0].name).to.equal(this.appName);
+					expect(apiKeys[0].id).to.equal(this.apiKey);
+				});
```

Comment:
```suggestion
				.then((apiKeys) => {
					expect(apiKeys).to.be.an('array');
					expect(apiKeys).to.have.lengthOf(1);
					expect(apiKeys[0].name).to.equal(null); // but it might be `''`, plz double check.
					expect(apiKeys[0].id).to.equal(this.apiKey.id);
				});
```
or convert to async-await